checks/signed_tags.go

```diff
@@ -61,10 +60,18 @@ func SignedTags(c checker.Checker) checker.CheckResult {
 			return checker.RetryResult(err)
 		}
 		if gt.GetVerification().GetVerified() {
-			c.Logf("signed tag found: %s, commit: %s", t.Name, sha)
+			c.Logf("verified tag found: %s, commit: %s", t.Name, sha)
```

Comment:
verified is slightly confusing word, that is why we keep check name as signed. i prefer signed for all of these, thoughts ?

Reply:
I changed the terminology for a repo such as git, as shown below.  They're signed but GH can't verify the key, so the check fails.  Logging "unsigned tag found" in this scenario is confusing.

I'd like to keep the new "reason" entry in the output either way, but I can change back to "signed" instead of "verified" if you wish.
```
go run . --repo=github.com/git/git --show-details --checks=Signed-Tags
Starting [Signed-Tags]
Finished [Signed-Tags]

RESULTS
-------
Signed-Tags: Fail 10
    !! unverified tag found: v2.29.0, commit: 0c6ff04c974f0aab239e9ba46307e1d1a4904d66, reason: unknown_key
    !! unverified tag found: v2.29.1, commit: 6c5034eca4973ae22eee0436e34f9f010895bea3, reason: unknown_key
    !! unverified tag found: v2.29.2, commit: 1ec19b7757a1acb11332f06e8e812b505490afc6, reason: unknown_key
    !! unverified tag found: v2.30.0-rc0, commit: a8eaf9de52c2d49799d7dc724e688ccbfa74390c, reason: unknown_key
    !! unverified tag found: v2.30.0-rc1, commit: cb70effebd91d9e0d4ce81650785e973b5d16ad1, reason: unknown_key
    found 0 out of 5 verified tags
```